grpc/src/main/java/io/stargate/grpc/service/CreateTableHandler.java

```diff
@@ -0,0 +1,225 @@
+import java.nio.ByteBuffer;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.concurrent.CompletionStage;
+
+class CreateTableHandler extends MessageHandler<CqlTableCreate> {
+
+  private final Persistence.Connection connection;
+  private final Persistence persistence;
+  private final String query;
+
+  CreateTableHandler(
+      CqlTableCreate createTable,
+      Persistence.Connection connection,
+      Persistence persistence,
+      StreamObserver<QueryOuterClass.Response> responseObserver) {
+    super(createTable, responseObserver);
+    this.connection = connection;
+    this.persistence = persistence;
+    this.query = buildQuery(createTable);
+  }
+
+  @Override
+  protected void validate() {
+    // intentionally empty
+  }
+
+  @Override
+  protected CompletionStage<QueryOuterClass.Response> executeQuery() {
+    long queryStartNanoTime = System.nanoTime();
+    try {
+      return connection
+          .execute(
+              new SimpleStatement(query),
+              makeParameters(connection.clientInfo()),
+              queryStartNanoTime)
+          .thenApply(result -> makeResponseBuilder(result).build());
+    } catch (Exception e) {
+      return failedFuture(e, false);
+    }
+  }
+
+  private String buildQuery(CqlTableCreate createProto) {
+    String keyspaceName =
+        persistence.decorateKeyspaceName(
+            createProto.getKeyspaceName(), GrpcService.HEADERS_KEY.get());
+    CqlTable tableProto = createProto.getTable();
+    DefaultCreateTable builder =
+        (DefaultCreateTable) SchemaBuilder.createTable(keyspaceName, tableProto.getName());
+    if (createProto.getIfNotExists()) {
+      builder = (DefaultCreateTable) builder.ifNotExists();
+    }
+    for (ColumnSpec columnProto : tableProto.getPartitionKeyColumnsList()) {
+      builder =
+          (DefaultCreateTable)
+              builder.withPartitionKey(columnProto.getName(), convertType(columnProto.getType()));
+    }
+    for (ColumnSpec columnProto : tableProto.getClusteringKeyColumnsList()) {
+      builder =
+          (DefaultCreateTable)
+              builder.withClusteringColumn(
+                  columnProto.getName(), convertType(columnProto.getType()));
+    }
+    for (ColumnSpec columnProto : tableProto.getStaticColumnsList()) {
+      builder =
+          (DefaultCreateTable)
+              builder.withStaticColumn(columnProto.getName(), convertType(columnProto.getType()));
+    }
+    for (ColumnSpec columnProto : tableProto.getColumnsList()) {
+      builder =
+          (DefaultCreateTable)
+              builder.withColumn(columnProto.getName(), convertType(columnProto.getType()));
+    }
+    for (Map.Entry<String, Schema.ColumnOrderBy> entry :
+        tableProto.getClusteringOrdersMap().entrySet()) {
+      builder =
+          (DefaultCreateTable)
+              builder.withClusteringOrder(entry.getKey(), convertOrder(entry.getValue()));
+    }
+    // TODO handle table options
```

Comment:
The driver's query builder uses strongly-typed options. If we pass in strings, they will get quoted.